### DISCUSSION TITLE
Arbitrary value and translate-x-[...] vs. -translate-x-[...]

### DISCUSSION THREAD

**Discussion by jensolafkoch** [tailwindlabs/tailwindcss#5406] [Help] [answered]
Hello,

it looks like I can use classes like:

translate-x-[100%] or translate-x-[1500px]

but not

-translate-x-[100%] or -translate-x-[1500px]

For the dashed (minus) variants of translate-x only the predefined TW classes seem to work, not the arbitrary value support?

**Comment by NovusOpus** [comment, marked as answer]
You can, but instead of putting the `-` before `translate` you put it with the value inside the Square brackets, like this: 
**`translate-x-[-100%] or translate-x-[-1500px]`**

**Reply by jensolafkoch** [reply]
Thanks. In fact quite obvious but sometimes you don't the forest for the trees ... :-)
